src/views/Affiliates/cards/AffiliateProgressCard.tsx

```diff
@@ -0,0 +1,53 @@
+import { AFFILIATES_REQUIRED_VOLUME_USD } from '@/constants/affiliates';
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+import { Output, OutputType } from '@/components/Output';
+
+export const AffiliateProgressCard = ({
+  volume = 0,
+  className,
+}: {
+  volume?: number;
+  className?: string;
+}) => {
+  const stringGetter = useStringGetter();
+
+  const progressPercent = ((volume / AFFILIATES_REQUIRED_VOLUME_USD) * 100).toFixed(0).toString();
```

Comment:
isn't `toString()` redundant after `toFixed()`

Reply:
USE OUTPUT and the associated formatting function